stockAPI.js

```diff
@@ -0,0 +1,24 @@
+const request = require('request');
+
+var stockSymbols = ["AAPL", "MSFT","HSBA.L"];
+var str = "";
+ 
+function getStockData(stockSymbolsArray){
+    stockSymbols.forEach(Symbol => {
```

Comment:
Javascript conventions are to use camel case https://chaseonsoftware.com/most-common-programming-case-types/#camelcase
so `Symbol` should be `symbol`